gprofiler/metadata/metadata_collector.py

```diff
@@ -33,4 +34,8 @@ def get_current_metadata(static_metadata: Metadata) -> Metadata:
     current_time = datetime.datetime.utcnow().replace(microsecond=0).isoformat()
     dynamic_metadata = static_metadata
     dynamic_metadata.update({"current_time": current_time})
+    # For now, just use the profiling mode provided in the args.
```

Comment:
This should go to `get_static_metadata` (line 28-29)

Reply:
I don't agree, now it's static but it may be dynamic in the future and thus it's better here.

Reply:
Pft. k